dev/benchmarks/complex_layout/pubspec.yaml

```diff
@@ -46,7 +46,7 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   test: 1.16.0-nullsafety.5
-  e2e: 0.7.0
+  e2e: 0.7.0+1
```

Comment:
Should this be `integration_test: ^0.9.2`? e2e is discontinued.

Reply:
I'm changing the these pubspec due to update-packages. I think they didn't migrated those tests yet.  I'm not sure which team is using them though :)